docs/admin/kube-scheduler.md

```diff
@@ -62,6 +62,10 @@ kube-scheduler
       --kube-api-burst=100: Burst to use while talking with kubernetes apiserver
       --kube-api-qps=50: QPS to use while talking with kubernetes apiserver
       --kubeconfig="": Path to kubeconfig file with authorization and master location information.
+      --leader-elect[=false]: Start a leader election client and gain leadership before executing scheduler loop. Enable this when running replicated schedulers.
+      --leader-elect-lease-duration=15s: Lease duration is the duration that non-leader candidates will wait until attempting to acquire leadership of a led but unrenewed leader slot. This is effectively the maximum duration that a leader can be stopped before it is replaced by another candidate. This is only applicable if leader election is enabled.
```

Comment:
s/Lease duration is the//

rewrite is much better -- only part people might find confusing is "will wait" -- will wait after what? "led but unrenewed" gives a good hint but maybe clearer to say "will wait after the most recent renewal" or something like that?